src/api/inSightApiClient.js

```diff
@@ -0,0 +1,27 @@
+import request from 'request';
+import Weather from '../models/weather';
+
+export default function callInSight() {
```

Comment:
I think this function name could be more descriptive - perhaps `getMarsWeatherData()`? What do you think?